src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -84,8 +85,55 @@ def get_next(self, sub_id):
             return None
         return self.arp_dest_list[right]
 
+class NextHopUpdater(MIBUpdater):
+    def __init__(self):
+        super().__init__()
+        self.db_conn = mibs.init_db()
+        self.update_data()
+
+    def update_data(self):
+        """
+        Update redis (caches config)
+        Pulls the table references for each interface.
+        """
+        self.nexthop_map = {}
+        self.route_list = []
+
+        self.db_conn.connect(mibs.APPL_DB)
+        route_entries = self.db_conn.keys(mibs.APPL_DB, "ROUTE_TABLE:*")
+
+        for route_entry in route_entries:
+            routestr = route_entry.decode()
+            ipnstr = routestr[len("ROUTE_TABLE:"):]
+            if ipnstr == "0.0.0.0/0":
+                ipn = ipaddress.ip_network(ipnstr)
+                ent = self.db_conn.get_all(mibs.APPL_DB, routestr, blocking=True)
```

Comment:
if blocking is true you need to check that ent is not None. I'd make it unblocking